### ISSUE TITLE
Implement personal chatting feature

### ISSUE THREAD

**Issue by Haider8** [Haider8/tmessage#16]
Right now only group messaging is available. There should be a feature which allows users to communicate one on one.

**Comment by brandonmaday** [comment]
I'd like to help with this one! Are you thinking of make this a feature for users who are online at the same time, or would you allow offline messages as well?

**Comment by Haider8** [comment]
Yes, you can work on this issue @brandonmaday :+1: . Currently, we are planning to add just the online feature to make the task simple as of now. Later we can work on the offline feature also.

Thank you :)

**Comment by Haider8** [comment]
@brandonmaday Hey, any progress on this issue?

**Comment by brandonmaday** [comment]
My apologies, work became a little crazy. I'm planning on working on this next week.   
  

  
  

  
  
>
